src/diamond/handler/stats_d.py

```diff
@@ -152,7 +154,7 @@ def _connect(self):
             self.connection = statsd.StatsClient(
                 host=self.host,
                 port=self.port
-            )
+            ).pipeline()
```

Comment:
Is there a reason to support both versions of the library? Seems like the former offers a superior API.